The patch adds Solaris specific information to the BSD manual pages.
Note that the userland mangler cannot be used since it does not
support mdoc format which is mainly used on FreeBSD operating systems.

The patch will not be sent to upstream
since this change is Solaris specific.

--- elixir-1.7.3/man/elixir.1.in	2018-10-25 07:37:36.228416485 +0000
+++ elixir-1.7.3/man/elixir.1.in	2018-10-25 07:36:36.740671693 +0000
@@ -1,3 +1,4 @@
+'\" te
 .Dd April 10, 2015
 .Dt ELIXIR 1
 .Os
@@ -38,6 +39,24 @@ Stores the magic cookie value which is u
 .Pq see above .
 If the file doesn't exist when a node starts, it will be created.
 .El
+
+.\" Oracle has added the ARC stability level to this manual page
+.Sh ATTRIBUTES
+See
+.Xr attributes 7
+for descriptions of the following attributes:
+.sp
+.TS
+box;
+cbp-1 | cbp-1
+l | l .
+ATTRIBUTE TYPE	ATTRIBUTE VALUE 
+=
+Availability	runtime/elixir
+=
+Stability	Pass-through volatile
+.TE 
+.Pp
 .Sh SEE ALSO
 .Xr elixirc 1 ,
 .Xr iex 1 ,
@@ -49,3 +68,10 @@ This manual page contributed by Evgeny G
 .It Main website: http://elixir-lang.org
 .It Documentation: http://elixir-lang.org/docs.html
 .El
+
+.Sh NOTES
+
+.\" Oracle has added source availability information to this manual page
+This software was built from source available at https://github.com/oracle/solaris-userland.  The original community source was downloaded from  https://github.com/elixir-lang/elixir/archive/v1.7.3/elixir-1.7.3.tar.gz
+
+Further information about this software can be found on the open source community website at https://elixir-lang.org/.
--- elixir-1.7.3/man/iex.1.in	2018-10-25 07:37:45.242173556 +0000
+++ elixir-1.7.3/man/iex.1.in	2018-10-25 07:36:36.741228061 +0000
@@ -1,3 +1,4 @@
+'\" te
 .Dd April 10, 2015
 .Dt IEX 1
 .Os
@@ -48,6 +49,24 @@ starts, it seeks the file
 .Pa .iex.exs
 and, in a case of success, executes the code from the file in the context of the shell. At first the search starts in the current working directory, then, if necessary, it continues in the home directory.
 .El
+
+.\" Oracle has added the ARC stability level to this manual page
+.Sh ATTRIBUTES
+See
+.Xr attributes 7
+for descriptions of the following attributes:
+.sp
+.TS
+box;
+cbp-1 | cbp-1
+l | l .
+ATTRIBUTE TYPE	ATTRIBUTE VALUE 
+=
+Availability	runtime/elixir
+=
+Stability	Pass-through volatile
+.TE 
+.Pp
 .Sh SEE ALSO
 .Xr elixir 1 ,
 .Xr elixirc 1 ,
@@ -59,3 +78,10 @@ This manual page contributed by Evgeny G
 .It Main website: http://elixir-lang.org
 .It Documentation: http://elixir-lang.org/docs.html
 .El
+
+.Sh NOTES
+
+.\" Oracle has added source availability information to this manual page
+This software was built from source available at https://github.com/oracle/solaris-userland.  The original community source was downloaded from  https://github.com/elixir-lang/elixir/archive/v1.7.3/elixir-1.7.3.tar.gz
+
+Further information about this software can be found on the open source community website at https://elixir-lang.org/.
--- elixir-1.7.3/man/elixirc.1	2018-10-25 07:38:10.209438738 +0000
+++ elixir-1.7.3/man/elixirc.1	2018-10-25 07:36:36.740980555 +0000
@@ -1,3 +1,4 @@
+'\" te
 .Dd April 10, 2015
 .Dt ELIXIRC 1
 .Os
@@ -39,6 +40,24 @@ Allows passing parameters to the Erlang
 Allows passing parameters to the Erlang compiler
 .Pq see Xr erlc 1 .
 .El
+
+.\" Oracle has added the ARC stability level to this manual page
+.Sh ATTRIBUTES
+See
+.Xr attributes 7
+for descriptions of the following attributes:
+.sp
+.TS
+box;
+cbp-1 | cbp-1
+l | l .
+ATTRIBUTE TYPE	ATTRIBUTE VALUE 
+=
+Availability	runtime/elixir
+=
+Stability	Pass-through volatile
+.TE 
+.Pp
 .Sh SEE ALSO
 .Xr elixir 1 ,
 .Xr iex 1 ,
@@ -50,3 +69,11 @@ This manual page contributed by Evgeny G
 .It Main website: http://elixir-lang.org
 .It Documentation: http://elixir-lang.org/docs.html
 .El
+
+
+.Sh NOTES
+
+.\" Oracle has added source availability information to this manual page
+This software was built from source available at https://github.com/oracle/solaris-userland.  The original community source was downloaded from  https://github.com/elixir-lang/elixir/archive/v1.7.3/elixir-1.7.3.tar.gz
+
+Further information about this software can be found on the open source community website at https://elixir-lang.org/.
--- elixir-1.7.3/man/mix.1	2018-10-25 07:38:20.341032429 +0000
+++ elixir-1.7.3/man/mix.1	2018-10-25 07:36:36.741499144 +0000
@@ -1,3 +1,4 @@
+'\" te
 .Dd May 27, 2015
 .Dt MIX 1
 .Os
@@ -134,6 +135,24 @@ Allows locking down the project dependen
 .It [1] http://www.erlang.org/doc/man/code.html#id103620
 .It [2] http://www.erlang.org/doc/design_principles/applications.html
 .El
+
+.\" Oracle has added the ARC stability level to this manual page
+.Sh ATTRIBUTES
+See
+.Xr attributes 7
+for descriptions of the following attributes:
+.sp
+.TS
+box;
+cbp-1 | cbp-1
+l | l .
+ATTRIBUTE TYPE	ATTRIBUTE VALUE 
+=
+Availability	runtime/elixir
+=
+Stability	Pass-through volatile
+.TE 
+.Pp
 .Sh SEE ALSO
 .Xr elixir 1 ,
 .Xr elixirc 1 ,
@@ -145,3 +164,11 @@ This manual page contributed by Evgeny G
 .It Main website: http://elixir-lang.org
 .It Documentation: http://elixir-lang.org/docs.html
 .El
+
+
+.Sh NOTES
+
+.\" Oracle has added source availability information to this manual page
+This software was built from source available at https://github.com/oracle/solaris-userland.  The original community source was downloaded from  https://github.com/elixir-lang/elixir/archive/v1.7.3/elixir-1.7.3.tar.gz
+
+Further information about this software can be found on the open source community website at https://elixir-lang.org/.
